collector.go

```diff
@@ -48,6 +55,7 @@ func ScrapeTarget(target string, config *Module) ([]gosnmp.SnmpPDU, error) {
 		p, err := strconv.Atoi(port)
 		if err != nil {
 			return nil, fmt.Errorf("Error converting port number to int for target %s: %s", target, err)
+			snmpCollectionErrors.Inc()
```

Comment:
This doesn't fit with the other two errors, as it's a user error rather than an snmp error.